app/models/result.rb

```diff
@@ -7,6 +7,8 @@ class Result < ApplicationRecord
   enum status: { fetching: 1, done: 2 }
 
   scope :by_keyword, lambda { |keyword|
-    where('keyword LIKE ?', '%' + keyword+ '%')
+    # rubocop:disable Style/StringConcatenation
+    where('keyword LIKE ?', '%' + keyword + '%')
```

Comment:
Does it fix this Rubocop error without disabling the rule? 🤔 

```suggestion
    where('keyword LIKE ?', "%#{keyword}%")
```

Reply:
Updated in f911c64 ✅